### PR TITLE
PCM-1313 : Hours Worked field for India associates

### PR DESCRIPTION
**Pull request by gandhikeyur**
@vrathee @ardasgup Please review

### CODE REVIEW

File: uds.js [hunk 1]
```diff
@@ -194,6 +194,14 @@
         var url = udsHostName.clone().setPath('/case/' + caseNumber + "/comments/public");
         return executeUdsAjaxCallWithData(onSuccess,onFailure,url,caseComment,'POST');
     };
+
+    uds.postPublicCommentsWithHours = function (onSuccess, onFailure, caseNumber, caseComment, hoursWorked) {
```
Comment (aritrod) [inline uds.js]: I think instead add condition inside postPublicComments method only no need of creating a separate method:
Something like - -
if(hoursWorked == undefined) {
call '/case/' + caseNumber + "/comments/public" }
else {
call '/case/' + caseNumber + "/comments/public/hoursWorked/"+hoursWorked }

Comment (aritrod) [inline uds.js]: And same thing for /private

**Comment by gandhikeyur** [comment]
@ardasgup @vrathee Code changed as per review comments. Kindly verify

**Comment by vrathee** [comment]
Reviewed.

**Comment by renujhamtani** [comment]
Looks good! Merging and then tagging!

**Pull request merged**